services/network/network.go

```diff
@@ -992,14 +988,14 @@ type PublicIPAddress struct {
 	*PublicIPAddressPropertiesFormat `json:"properties,omitempty"`
 	// Etag - A unique read-only string that changes whenever the resource is updated.
 	Etag *string `json:"etag,omitempty"`
-	// Zones - A list of availability zones denoting the IP allocated for the resource needs to come from.
-	Zones *[]string `json:"zones,omitempty"`
 	// ID - Resource ID.
 	ID *string `json:"id,omitempty"`
 	// Name - READ-ONLY; Resource name.
 	Name *string `json:"name,omitempty"`
 	// Type - READ-ONLY; Resource type.
 	Type *string `json:"type,omitempty"`
+	// Version
+	Version *string `json:"version,omitempty"`
```

Comment:
>Version *string `json:"version,omitempty"` [](http://example.com/codeflow?start=1&length=42)

this too, as far as i see its PublicIPAddressPropertiesFormat->PublicIPAddressVersion

Reply:
Version is not the IP version. I believe you or your team asked to have resource version for all. This was specifically asked. 

PublicIPAddressPropertiesFormat contains the pip ip address version.
type PublicIPAddressPropertiesFormat struct {
......
    // PublicIPAddressVersion - The public IP address version. Possible values include: 'IPv4', 'IPv6'
    PublicIPAddressVersion IPVersion `json:"publicIPAddressVersion,omitempty"`